docs/concepts/fields.md

```diff
@@ -1,29 +1,81 @@
 ??? api "API Documentation"
     [`pydantic.fields.Field`][pydantic.fields.Field]<br>
 
-The [`Field`][pydantic.fields.Field] function is used to customize and add metadata to fields of models.
+In this section, we will go through the available mechanisms to customize Pydantic model fields:
+default values, JSON Schema metadata, constraints, etc.
+
+To do so, the [`Field()`][pydantic.fields.Field] function is used a lot, and behaves the same way as
+the standard library [`field()`][dataclasses.field] function for dataclasses:
+
+```python
+from pydantic import BaseModel, Field
+
+
+class Model(BaseModel):
+    field: str = Field(frozen=True)
+```
+
+!!! note
+    Even though `field` is assigned a value, it is still required and has no default value. If you want
+    to emphasize on the fact that a value must be provided, you can use the [ellipsis][Ellipsis]:
+
+    ```python
+    class Model(BaseModel):
+        field: str = Field(..., frozen=True)
+    ```
+
+    However, its usage is discouraged as it doesn't play well with type checkers.
```

Comment:
This replaces the "required fields" section in the `models.md` page, which wasn't really useful imo (it is already stated in the "Basic model usage" section at the very beginning, and it's also pretty straightforward).